src/cljc/quil/core.cljc

```diff
@@ -1529,7 +1547,8 @@
   completed (or after setup completes if called during the setup
   method). "
   []
-  (.exit (ap/current-applet)))
+  #?(:clj (.exit (ap/current-applet))
+     :cljs (.remove (ap/current-applet))))
```

Comment:
I believe this broke cljs test page. We call `(q/exit)` at the end of each snippet: https://github.com/quil/quil/blob/master/test/cljs/quil/snippet.cljs#L37 and now it seems like p5js removes canvas even before we can see it. 

Should we just not call `(q/exit)` after each snippet? Don't know what's the proper way to reuse the same for multiple snippets in p5js.

Reply:
That's what I had done while [drafting the changes](https://github.com/anthonygalea/quil/commit/e92c6df521d75859fe01ad0b303b8989b019a2a8#diff-8a5b0cf307b160a77d1d44948df43dce). I was just removing any old canvases before a test is run. Works for both single and multiple. What do you think about that approach?